sktime/forecasting/var.py

```diff
@@ -25,8 +25,25 @@ class VAR(_StatsModelsAdapter):
         Estimation method to use
     verbose : bool (default = False)
         Print order selection output to the screen
+    trend : str {"c", "ct", "ctt", "n"}
+        "c" - add constant
+        "ct" - constant and trend
+        "ctt" - constant, linear and quadratic trend
+        "n" - co constant, no trend
+        Note that these are prepended to the columns of the dataset.
     missing: str, optional (default='none')
         A string specifying if data is missing
+    freq: str, tuple, datetime.timedelta, DateOffset or None, optional
+    dates: array_like
+        Must match number of rows of endog
+        A frequency specification for either `dates` or the row labels from
+        the endog / exog data.
+    ic: {'aic', 'fpe', 'hqic', 'bic', None}
+        Information criterion to use for VAR order selection.
+        aic : Akaike
+        fpe : Final prediction error
+        hqic : Hannan-Quinn
+        bic : Bayesian a.k.a. Schwarz
```

Comment:
could you kindly mention there also the default values? e.g. `optional, default=None`

Reply:
seems also for other params that was not fully done, do you mind to check that for all params? :)

Reply:
No problem! I added default values for all of them. I found the description of `freq` was misplaced in `dates`, so that was properly corrected as well. Thank you for your patience :)